services/submission/src/main/java/app/coronawarn/server/services/submission/verification/TanVerifier.java

```diff
@@ -88,6 +88,7 @@ private boolean checkTanSyntax(String tan) {
       UUID.fromString(tan);
       return true;
     } catch (IllegalArgumentException e) {
+      logger.debug("UUID creation failed for value: " + tan, e);
```

Comment:
Preferrably ``logger.debug("UUID creation failed for value: {}", tan, e);``

@MKusber is also touching this line in #319